### PR TITLE
docs: Add example to useHandler docs

### PR DESCRIPTION
**Pull request by patrycjakalinska**
Few copy changes and added an example to [useHandler](https://docs.swmansion.com/react-native-reanimated/docs/advanced/useHandler/) docs.

image

### CODE REVIEW

File: packages/docs-reanimated/docs/advanced/useHandler.mdx [hunk 1]
```diff
@@ -91,7 +83,9 @@ Each of the event worklets will receive the following parameters when called:
 
 #### `dependencies` <Optional />
 
-Changes in `dependencies` array cause `useHandler` hook to receive updated values during rerender of the wrapping component. This matters when, for instance, worklet uses values dependent on the component's state.
+An optional array of dependencies.
+
+Only relevant when using Reanimated [without the Babel plugin on the Web.](/docs/guides/web-support/#web-without-the-babel-plugin)
 
 This array may contain:
```
Comment (kacperkapusciak) [inline packages/docs-reanimated/docs/advanced/useHandler.mdx:91]: We can remove this and the following paragraph too

**Review by kacperkapusciak — APPROVED** [review]
approve in blanco but let's get rid of the `any` in the example code first

File: packages/docs-reanimated/docs/advanced/useHandler.mdx [hunk 2]
```diff
@@ -105,27 +99,10 @@ The hook returns a context that will be reused by event handlers and value that
 
 ## Example
```
Comment (kacperkapusciak) [inline packages/docs-reanimated/docs/advanced/useHandler.mdx:101]: I'd also add a single sentence here that this example is easier to implement with `useScrollViewOffset`

File: packages/docs-reanimated/docs/advanced/useHandler.mdx (hunk 1 above)
Comment (patrycjakalinska) [reply packages/docs-reanimated/docs/advanced/useHandler.mdx:91]: Whole info about contents of array, si?

File: packages/docs-reanimated/src/examples/UseHandlerEventExample.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+} from 'react-native-reanimated';
+import { TextInput, SafeAreaView, View, StyleSheet } from 'react-native';
+
+const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
+
+function UseHandlerExample() {
+  const offsetY = useSharedValue(0);
+
+  const handlers = {
+    onScroll: (event: any) => {
```
[the remark this answers is not in the document]
Comment (tjzel) [reply packages/docs-reanimated/src/examples/UseHandlerEventExample.tsx]: Import `ScrollEvent` type from RN 👍

**Pull request merged**